assets/src/scripts/monitoring-location/components/hydrograph/index.js

```diff
@@ -164,6 +167,70 @@ export const attachToNode = function(store,
 
                 nodeElem.select('.ts-legend-controls-container')
                     .call(drawGraphControls, store);
+
+                // Construct and add the hrefs needed so users can download the data corresponding to the currently displayed hydrograph with the 'download data' links
+                nodeElem.select('#iv-download-container').call(link(store, (container, {currentIVDateRange, parameterCode, showIVTimeSeries, queryInformation}) => {
```

Comment:
I would like this to follow the same pattern that we have already established. Put the rendering code in a separate module and follow the pattern already established for other sub-components. https://github.com/usgs/waterdataui/blob/master/assets/src/scripts/monitoring-location/components/hydrograph/method-picker.js is a nice simple example. The code will be simpler, less prone to errors as things change, and it can be more thoroughly tested. Let's talk if you have questions or comments.

Reply:
Makes sense, I'll fix it up.